projects/ad4630_fmc/zed/Makefile

```diff
@@ -4,13 +4,11 @@
 ## Auto-generated, do not modify!
 ####################################################################################
 
-PROJECT_NAME := ad4630_fmc_zed
+PROJECT_NAME := ad4x3x_fmc_zed
 
-M_DEPS += system_constr_8sdi.xdc
-M_DEPS += system_constr_4sdi.xdc
-M_DEPS += system_constr_2sdi.xdc
-M_DEPS += system_constr_1sdi.xdc
-M_DEPS += ../common/ad463x_bd.tcl
+M_DEPS += system_constr.xdc
```

Comment:
Regenerate the Makefile, as system_constr.xdc and system_constr.tcl shouldn't be added here.

Reply:
Done!